docs_src/design/adr/devops/0007-Release-Automation.md

```diff
@@ -56,32 +60,19 @@ Go modules, Application and Device SDKs only release a GitHub tag as their relea
 
 The release automation for go modules is used to set a new version that increases the major or minor version for the module. (IE: 1.0.0 -> 1.1.0) For the Application and Device SDKs it is used to set the final release version. (IE: 1.0.0-dev.X -> 1.0.0)
 
-### Application, Device Services and Supporting Docker Images
+### Core Services, Application Services, Device Services and Supporting Docker Images
```

Comment:
What about security and system management services, do we consider these part of "Core"?

Reply:
Lisa considered everything outside of app services and device services part of "core".  Supporting services to her were things like Redis or Mongo.  Might be worth expanding this to define core services or say core services, supporting services, system management services, ...

Reply:
I updated with the suggested clarification.